app/constants.js

```diff
@@ -46,4 +46,7 @@ module.exports = {
   DEFAULT_LOGO_WIDTH: 64,
   DEFAULT_HEADER_HEIGHT: 96,
   DEFAULT_HEADER_WIDTH: 300,
+  MASTER_ACCESS: ['admin', 'manager'],
```

Comment:
Just to double check- this means master access to a report, right? So the managers wouldn't have access to admin routes still

Reply:
Report managers will has master access to all of the reports from the projects they have access to. However, they will not have master access to non-report routes (i.e projects, templates, users, etc.) like managers and admin have. They will also not have access to admin only routes. I describe it a bit more in a comment on this thicket DEVSU-1617.

If you are talking about managers having access to admin only routes. They do not. Most admin only routes are in the SPECIAL_CASES section of the acl and will only allow users to perform the request if they are an admin.